openpype/hosts/fusion/deploy/MenuScripts/install_pyside2.py

```diff
@@ -0,0 +1,14 @@
+# This is just a quick hack for users running Py3 locally but having no
+# Qt library installed
+import os
+
+try:
+    print("Qt library found, nothing to do.")
+    from Qt import QtWidgets
```

Comment:
I would do the print after import.
```suggestion
    from Qt import QtWidgets
    print("Qt library found, nothing to do.")
```